EmployeeDirectory.UI/Components/Pages/SearchBox.razor

```diff
@@ -0,0 +1,46 @@
+﻿@code {
+    [Parameter]
+    public string Placeholder { get; set; } = "Search...";
+
+    [Parameter]
+    public EventCallback<List<EmployeeDTO>> OnSearchChanged { get; set; }
+
+    [Parameter]
+    public IEnumerable<EmployeeDTO> filteredTasks { get; set; }
+
+    [Parameter]
+    public IEnumerable<EmployeeDTO> tasks1 { get; set; }
+
+
+
+    private string searchText = string.Empty;
+
+    private async Task OnInput(ChangeEventArgs e)
+    {
+        searchText = e.Value.ToString();
+        ApplyFiltersAndSort();
+        await OnSearchChanged.InvokeAsync(filteredTasks.ToList());
+    }
+
+
+
+
+    private void ApplyFiltersAndSort()
```

Comment:
where are we doing Sort?

Reply:
Actually this component is taken from previous task, I thought to remove the unnecessary functions once it starts working fine.